src/main/java/com/fasttime/domain/comment/controller/CommentController.java

```diff
@@ -0,0 +1,54 @@
+package com.fasttime.domain.comment.controller;
+
+import com.fasttime.domain.comment.dto.request.CreateCommentRequest;
+import com.fasttime.domain.comment.dto.request.DeleteCommentRequest;
+import com.fasttime.domain.comment.dto.request.UpdateCommentRequest;
+import com.fasttime.domain.comment.service.CommentService;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Slf4j
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/v1/comment")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @PostMapping("/create")
+    public ModelAndView create(@Valid @RequestBody CreateCommentRequest createCommentRequest,
+        ModelAndView mv) {
+        log.info("CreateCommentRequest: " + createCommentRequest);
+        mv.setStatus(HttpStatus.CREATED);
+        mv.addObject(commentService.createComment(createCommentRequest));
+        mv.setViewName("registerCommentSuccess");
+        return mv;
+    }
```

Comment:
`nit` : `@Controller` 이 사용되면서 `ModelAndView` 는 잘 사용되지 않는 추세입니다. 해당 에너테이션을 사용하면서 내부적으로 model과 view를 연결해주고 있기 때문인데요. 덕분에 굳이 viewName을 설정하지 않아도 그냥 viewName을 리턴하는 것 만으로도 mdoel을 넘겨줄 수 있습니다. 위 코드는 이렇게 해결이 가능하겠네요!
```suggestion
    @ResponseStatus(HttpStatus.CREATED)
    public String create(@Valid @RequestBody CreateCommentRequest createCommentRequest, Model model) {
        log.info("CreateCommentRequest: " + createCommentRequest);
        model.setStatus(HttpStatus.CREATED);
        model.addAttribute("createdComment", commentService.createComment(createCommentRequest));
        return "registerCommentSuccess";
    }
```

Reply:
저도 이전에 저렇게 쓰다가 현업에 계신 분이 ModelAndView로 보내는 편이라고 하셔서 쓰기 시작했어요! 찾아보니 ModelAndView를 쓰는 방식이 좀 더 구식이라고 나오네요! 바꿔보도록 할게요. 감사합니다 😀

Reply:
그렇군요 😮 이거 번데기 앞에서 주름을 잡은 셈이 되네요 ㅋㅋㅋㅋ
생각해보니 동일한 attribute로 여러 `status`, `viewname`을 설정할 수 있다는 점이 매력적일 수 있겠네요